js/taskOverlay.js

```diff
@@ -54,62 +54,76 @@ function getTaskOverlayTabElement (tab, task) {
   return item
 }
 
-function getTaskElement (task, taskIndex) {
-  var container = document.createElement('div')
-  container.className = 'task-container'
+function addTabCloseButton (tabContainer, taskTabElement) {
+  var closeTabButton = document.createElement('button')
+  closeTabButton.innerHTML = '✕'
+  closeTabButton.className = 'closeTab'
 
-  container.setAttribute('data-task', task.id)
+  closeTabButton.addEventListener('click', function (e) {
+    closeTab(taskTabElement.getAttribute('data-tab'))
+    tabContainer.removeChild(taskTabElement)
 
-  var taskActionContainer = document.createElement('div')
-  taskActionContainer.className = 'task-action-container'
+    // do not close taskOverlay
+    e.preventDefault()
```

Comment:
```preventDefault``` doesn't do the correct thing here - it will stop the browser from doing anything based on this event, but it won't stop other listeners from running. ```stopImmediatePropogation``` should work instead.